wasm/store.go

```diff
@@ -431,7 +439,18 @@ func (s *Store) buildFunctionInstances(module *Module, target *ModuleInstance) (
 			return rollbackFuncs, fmt.Errorf("code index out of range")
 		}
 
+		// Get the function name if it exists.
+		// Please note that we plus with importedFunctionNum as
+		// the index in the function name section is for all the declarations,
+		// meaning that index starts with imported functions, and then
+		// the function defined in the module comes.
+		name, ok := functionNames[uint32(codeIndex)+uint32(importedFunctionNum)]
+		if !ok {
+			name = "unknown"
```

Comment:
if empty isn't a valid function name, let's use that vs "unknown"

Reply:
empty is valid but this block means that `functionNames` map is empty (i.e. name custom section is removed out), so I think I will keep this as-is!